Add registers for Pool 40 accessory and some more missing registers to S1156/S1256

vtamm opened the pull request.

yozik04 commented on nibe/data/extensions.json:
```diff
@@ -615,5 +615,16 @@
+    ],
+    "data": {
+      "40061": {
+        "default": 430.0
```
Why you need this default? I do not think we use it anywhere?

vtamm replied:
@yozik04 Hey! Apologies for submitting a so far undocumented PR, I am going to write a description but did not have time for it yesterday – will update it today!

Regarding this change in particular: the CSV export from my S1256 included this change in the default value of this register compared to the CSV file in the repository. I wrote in the commit description that I suspect this change might come from the latest firmware (3.4.11). But it also seems to me like it is a pretty insignificant change, and for that reason maybe it can be ignored? (If you agree, I can remove the addition to `extensions.json` from this PR.)

yozik04 replied:
Just ignore it. It is not possible to make it perfect. It changes every month for S series.

vtamm commented on nibe/data/s1156_s1256.csv:
```diff
@@ -1173,6 +1188,7 @@ Heating, auto	MODBUS_HOLDING_REGISTER	3059	1		4	0	1	1
 Factor	MODBUS_HOLDING_REGISTER	3033	1		4	1	10	5
 id:10881	MODBUS_HOLDING_REGISTER	3062	1		4	0	1	0
 id:10890	MODBUS_HOLDING_REGISTER	3063	1		4	0	1	0
+Hot water start (BT5)	MODBUS_INPUT_REGISTER	2014	10	°C	2	0	0	0
```
@yozik04 regarding this line and why it is "new": I see that [you had a discussion recently](https://github.com/yozik04/nibe/discussions/196) about adding this register. It was then added to `s1155_s1255.csv` in #197 but then it seems the "Hot water start" was lost for the xx56 models when the new `s1156.csv` file was added in #198 and later renamed to `s1156_s1256.csv` in [this commit](2a41278bfce998ac426c2a68f44883ed48e71c1d).

vtamm commented:
Hey @yozik04, I've updated the PR and removed the addition to `extensions.json` as well as added four more new registers which showed up in my CSV export today after I connected two THS 10 room temperature sensors. That's all I got for now!

yozik04 commented:
@vtamm Thank you for the effort!

vtamm commented:
@yozik04 this is unrelated to this PR but for lack of a better place I'll notify you here; I've just started looking into the mapping of the S-series climate systems properties in `coil_groups.py` to the corresponding registers and it looks like some of them might be mapped to the wrong value/register.

I'm reviewing all of the climate systems now and will submit a new PR with any changes probably in the next couple of hours. Just wanted to let you know right away in case you're planning to tag a new version soon – you might want to hold it until I'm done with this review so that you can include the corrected mappings in the next release!

vtamm commented:
@yozik04 if you don't mind, I have a couple of things I'd like to ask/discuss with you which are better suited for chat, so I sent you a friend request (Flux) on the Home Assistant Discord server!

yozik04 commented:
I've added you, but I think you want to talk to @elupus instead about S-series.
